src/renderer/components/CustomEdge/styles.scss

```diff
@@ -0,0 +1,74 @@
+@keyframes dashdraw-chain {
+    from {
+        stroke-dashoffset: 13;
+    }
+}
+
+:not(.no-chain-animation) .edge-chain.running {
+    animation: dashdraw-chain 0.5s linear infinite !important;
+}
+
+.no-chain-animation .react-flow__edge.running path {
+    animation: none !important;
+}
+
+.edge-chain-links {
+    stroke-width: 2px;
+    transition-duration: 0.15s;
+    transition-property: stroke-width, stroke;
+    transition-timing-function: ease-in-out;
+    cursor: pointer;
+    stroke-dasharray: 0 !important;
+
+    &.hovered {
+        stroke-width: 4px;
+    }
+
+    &.animated {
+        stroke-dasharray: 0 !important;
+    }
+}
+
+.edge-chain {
+    stroke-width: 6px;
+    transition-duration: 0.15s;
+    transition-property: stroke-width, stroke;
+    transition-timing-function: ease-in-out;
+    cursor: pointer;
+    stroke-dasharray: 1 10 !important;
+    stroke-dashoffset: 2;
+    stroke-linecap: round;
+    animation: 'none';
+    opacity: 0%;
+
+    &.animated {
+        stroke-dasharray: 1 10 !important;
+    }
+
+    &.hovered {
+        stroke-width: 8px;
+    }
+
+    &.running {
+        animation: 'dashdraw-chain 0.5s linear infinite';
+        opacity: 100%;
+    }
+
+    &.dot {
+        stroke: var(--chain-hole-color) !important;
```

Comment:
Since you are using colors, could you add `@use 'colors.scss';`?

Reply:
Is that needed? This works as is

Reply:
Yes, it works because global.scss imports color.scss and global.scss happens to be present. However, this is global reasoning. There's no way to tell that a variable called `--chain-hole-color` exists just by looking at this file. This not only makes it more nebulous where this variable comes from, it will also stop working if we remove the color.scss import in global.scss (e.g. because we moved out most rules into dedicated files and don't need the import anymore).